packages-exp/auth-exp/src/core/user/token_manager.ts

```diff
@@ -54,14 +55,6 @@ export class StsTokenManager {
     response: IdTokenResponse | FinalizeMfaResponse
   ): void {
     _assert(response.idToken, AuthErrorCode.INTERNAL_ERROR);
-    _assert(
```

Comment:
maybe keep these assertions and additionally assert that if things are undefined then we are in pass through mode?

Reply:
Good q - so regarding the first part (and please correct me if I'm wrong!), the set of idTokens that would fail the first assertion (L56) is a superset of the set of idTokens that would fail the second assertion (L57-L60), so the second assertion can be removed safely. For the second part of the question and regarding the third assertion (L61-L64), we know that we're in passthrough mode when there is no refresh token returned and to account for that newly valid case, we need to remove the assert - not sure if there's another way of additionally asserting we're in passthrough mode you're referring to?

Reply:
@lisajian I think your assessment is right here. The second assertion is definitely redundant and there's no other indicator for being in passthrough mode as far as I can tell

Reply:
ack